domains/DataCollection/index.js

```diff
@@ -4,25 +4,49 @@ import {
   StyleSheet, Text, ScrollView, View
 } from 'react-native';
 
+import {
+  Button
+} from 'react-native-paper'
+
 import Forms from './Forms';
 
 export default function DataCollection({ navigation }) {
   const [scrollViewScroll, setScrollViewScroll] = useState();
+
+  const [showForms, setShowForms] = React.useState(false);
+  const [selectForm, setSelectForm] = React.useState('id');
+
   return (
     <View
       style={styles.container}
       onStartShouldSetResponderCapture={() => {
         setScrollViewScroll(true);
       }}
     >
-      <ScrollView style={styles.container} keyboardShouldPersistTaps="always" scrollEnabled={scrollViewScroll}>
-        <Text style={styles.line}>Welcome to the data collection page.</Text>
-        <Forms
-          style={styles.line}
-          navigation={navigation}
-          scrollViewScroll={scrollViewScroll}
-          setScrollViewScroll={setScrollViewScroll}
-        />
+      <ScrollView keyboardShouldPersistTaps="always" scrollEnabled={scrollViewScroll}>
+        {!showForms
+          && (
+            <View>
+              <Text style={styles.line} onPress={() => setShowForms(true)}>New Record</Text>
+              <Text style={styles.line}>Find Record</Text>
+              <Text style={styles.line}>New Asset</Text>
+              <Text style={styles.line}>Find Asset</Text>
+            </View>
+          )}
+        {showForms
+          && (
+            <View>
+              <Button onPress={() => setShowForms(false)}>Back to Data Collection Screen</Button>
```

Comment:
this takes them back to the main data collection screen

New records
Find Records
new Assets
find assets